### ISSUE TITLE
[TypeScript] Add React properties

### ISSUE THREAD

**Issue by CharlesStover** [CharlesStover/reactn#25]
Currently only ReactN properties are importable: decorator, addReducer, setGlobal, Component, PureComponent, etc.

React properties should be importable as well: createElement (though this doesn't break JSX for some reason?), useEffect, useState

In index.d.ts, we have:
```
interface ReactN {
  importableProperty: Etc;
}
```

It needs to be:
```
interface ReactN extends React {
  importableProperty: Etc;
}
```

However, this throws `Cannot use namespace 'React' as a type`, since React is not an object but a namespace.

Unsure how to fix.

**Comment by Fsarmento** [comment]
Have you tried changing from `declare const _: ReactN;` to `declare const _: ReactN & typeof React;` [here](https://github.com/CharlesStover/reactn/blob/9682e3cf8fe11eb34391c7ea2f6d86f3fcd722f8/index.d.ts#L113)?

**Comment by CharlesStover** [comment]
Thank you for the suggestion and sorry for the late reply. I have not had time to test this, but I will. It's been in the back of my mind since you commented, but I just haven't been able to set down and test TypeScript definitions in a while. 😅

**Comment by CharlesStover** [comment]
As an update, this is fixed in the upcoming 2.0 release. I think `typeof` was the issue.
